### DISCUSSION TITLE
Glitter link doesn't seem to be working.

### DISCUSSION THREAD

**Discussion by srijan-paul** [flix/flix#6921] [Q&A] [answered]
Hi,

I've contributed to some research programming languages in the past ([Pallene](https://github.com/pallene-lang/)), and have been looking for a decent functional language with effects. I was almost about to write a language of my own when I found Flix.
I love the ideas that have gone into developing it, and would like to hack on the compiler. Mostly so I can gauge the amount of work that goes into a type system and codegen for a language of Flix's caliber, and to get more familiar with the language itself – since the compiler is bootstrapped.

Is this discussions panel now the primary mode of discourse for the Flix compiler?
The Glitter link 404s for me for some reason :/
<img width="976" alt="SCR-20231227-khuc" src="https://github.com/flix/flix/assets/50487716/11c21590-2dac-463f-a789-c325c116293c">

**Comment by magnus-madsen** [comment, marked as answer]
Hi @srijan-paul and thanks for your interest!

Where do you see the link?

Does this link work: https://app.gitter.im/#/room/#flix_Lobby:gitter.im

**Comment by srijan-paul** [comment]
Thanks, @magnus-madsen.
It seems that my VPN is messing with gitter. Haven't seen this before.
